src/lib/nameduser/NamedUserTagRequest.ts

```diff
@@ -26,7 +39,19 @@ export class NamedUserTagRequest implements IRequest {
   }
 
   getRequestBody(): string {
-    return JSON.stringify(this)
+    const payload: IPayload = {
+      audience: this.audience,
+    }
+
+    if (Object.keys(this.add).length) {
+      payload['add'] = this.add
+    }
+
+    if (Object.keys(this.remove).length) {
+      payload['remove'] = this.remove
+    }
```

Comment:
I didn't see anything in the documentation mentioning if you can send these as an empty object if no tags are added for `add` or `remove`, but we know it works when you omit them, so I'm only adding them if tags have been added to one or both.

Reply:
It does seem weird that someone could send both add and remove, but at the same time, that's kind of on them if they do something that silly :)

Reply:
I could possibly see a use case to where you have tags you want to add for one user and remove for another all within the same request.

Reply:
Are they both POST calls? I assumed removal of tags was a `DELETE` but yeah looking at the docs, guess it's possible. I keep forgetting they don't treat TAGS as a resource, which is *extremely* annoying.

Reply:
Yeah, just a POST call to `/named_users/tags`, with either or both `remove` and `add` properties